From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Spottedleaf <Spottedleaf@users.noreply.github.com>
Date: Fri, 12 Jun 2020 13:33:19 -0700
Subject: [PATCH] Fix sand duping

If the falling block dies during teleportation (entity#move), then we need
to detect that by placing a check after the move.

I removed the patch

diff --git a/src/main/java/net/minecraft/server/EntityFallingBlock.java b/src/main/java/net/minecraft/server/EntityFallingBlock.java
index 0fbcf454c2db2427055123424c6d10cfbb939c17..cc0f54386bbce27dda4d720dba07b2b6b6fd912b 100644
--- a/src/main/java/net/minecraft/server/EntityFallingBlock.java
+++ b/src/main/java/net/minecraft/server/EntityFallingBlock.java
@@ -64,6 +64,9 @@ public class EntityFallingBlock extends Entity {
 
     @Override
     public void tick() {
+        // Paper start - fix sand duping
+        //removed
+        // Paper end - fix sand duping
         if (this.block.isAir()) {
             this.die();
         } else {
@@ -86,6 +89,10 @@ public class EntityFallingBlock extends Entity {
 
             this.move(EnumMoveType.SELF, this.getMot());
 
+            // Paper start - fix sand duping
+            //removed
+            // Paper end - fix sand duping
+
             // Paper start - Configurable EntityFallingBlock height nerf
             if (this.world.paperConfig.fallingBlockHeightNerf != 0 && this.locY() > this.world.paperConfig.fallingBlockHeightNerf) {
                 if (this.dropItem && this.world.getGameRules().getBoolean(GameRules.DO_ENTITY_DROPS)) {
